<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Storage Facilities Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_data">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>500</width>
      <height>250</height>
     </rect>
    </property>
    <property name="title">
     <string>Данные для запросов</string>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_id">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_id"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_address">
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_address"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_type">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_type"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_capacity">
       <property name="text">
        <string>Capacity:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_square">
       <property name="text">
        <string>Square:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_main">
       <property name="text">
        <string>Main of Storage:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_main"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_capacity"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_square"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_actions">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>300</width>
      <height>250</height>
     </rect>
    </property>
    <property name="title">
     <string>Запросы</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_get">
       <property name="text">
        <string>GET</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_post">
       <property name="text">
        <string>POST</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_patch">
       <property name="text">
        <string>PATCH</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_delete">
       <property name="text">
        <string>DELETE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_list">
       <property name="text">
        <string>LIST</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
